less/visua11ySettings.less

```diff
@@ -1,21 +1,114 @@
-// Notify override
+// Notify overrides
 // --------------------------------------------------
 .notify__popup.is-visua11ysettings {
   background-color: transparent;
 
   .notify__popup-inner {
-    max-width: @device-width-small;
+    max-width: @device-width-medium;
     margin: auto;
     background-color: @notify;
     padding: @item-padding (@item-padding / 2);
     border-radius: 3px;
   }
 }
 
-// UI settings
+// Notify view
 // --------------------------------------------------
-.visua11ysettings {
+.notify__popup.is-visua11ysettings {
```

Comment:
This selector `.notify__popup.is-visua11ysettings`, should be replaced with `.visua11ysettings-notify` once added (for consistency with drawer selector `.visua11ysettings-drawer`).

Reply:
I've committed this change.